site/src/middleware.ts

```diff
@@ -101,7 +106,7 @@ export async function middleware(request: NextRequest) {
 
 type RewritesMap = Map<string, Rewrite>;
 
-async function createRewrites(): Promise<RewritesMap> {
+async function createRewrites(scope: Pick<GQLRedirectScope, "domain">): Promise<RewritesMap> {
```

Comment:
```suggestion
async function createRewrites(scope: GQLRedirectScope): Promise<RewritesMap> {
```

Pick shouldn't be necessary here.

Reply:
https://github.com/vivid-planet/comet-starter/pull/315/commits/7fb53030b9e33dfe2d8b7f4b6ff1be5dfd05cb34